addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -318,6 +338,14 @@ function factory(dependencies) {
         managerVisual: attr({
             related: 'manager.visual',
         }),
+        // chatWindow position
```

Comment:
Usually use multi-line comment for field description:
```js
/**
 *
 */
```

And description is too short and misleading. At least should explain "visible" and "index" (not `visible: [...], index: [...]`, that's boring and bad way to describe in this context)